go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -95,7 +95,7 @@ func (api *EthereumAPI) GetTransactionReceipt(_ context.Context, encryptedParams
 }
 
 // EstimateGas requests the enclave the gas estimation based on the callMsg supplied params (encrypted)
-func (api *EthereumAPI) EstimateGas(_ context.Context, encryptedParams common.EncryptedParamsEstimateGas, _ *rpc.BlockNumberOrHash) (*string, error) {
+func (api *EthereumAPI) EstimateGas(_ context.Context, encryptedParams common.EncryptedParamsEstimateGas) (*string, error) {
```

Comment:
`_ *rpc.BlockNumberOrHash` is encrypted in the `common.EncryptedParamsEstimateGas`